frontend/src/components/WhosMyAdministrator.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { Button, Popover } from '@patternfly/react-core';
+import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+import PopoverListContent from '~/components/PopoverListContent';
+import { FindAdministratorOptions } from '~/pages/projects/screens/projects/const';
+
+type Props = {
+  buttonLabel?: string;
+  headerContent?: string;
+  leadText?: string;
+  isInline?: boolean;
+  contentTestId?: string;
+  linkTestId?: string;
+};
+
+const WhosMyAdministrator: React.FC<Props> = ({
+  buttonLabel = "Who's my administrator?",
+  headerContent,
+  leadText,
+  isInline,
+  contentTestId,
+  linkTestId,
+}) => (
+  <Popover
+    showClose
+    position="bottom"
+    headerContent={headerContent || 'Your administrator might be:'}
+    hasAutoWidth
+    maxWidth="370px"
+    bodyContent={
+      <PopoverListContent
+        data-testid={contentTestId}
+        leadText={leadText}
+        listHeading={headerContent ? 'Your administrator might be:' : undefined}
+        listItems={FindAdministratorOptions}
+      />
+    }
+  >
+    <Button
+      isInline={isInline}
+      variant="link"
+      icon={<OutlinedQuestionCircleIcon />}
+      data-testid={linkTestId}
+    >
+      {buttonLabel}
+    </Button>
```

Comment:
Requires an `aria-label` if there's no button label.

Likely always need a label prop and add a separate prop to hide the label.

Reply:
Must have made the comment before realizing there was also a default value. So the intent is to always have a label.